### PR TITLE
Update ABSTRACT CB.yaml

### PR DESCRIPTION
**Pull request by RathishKJ**

### CODE REVIEW

**Review by tasodorff — CHANGES_REQUESTED** [review]
Please make minor changes.

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 1]
```diff
@@ -470,6 +470,12 @@ SS:
   - powerfactor_sensor
   - current_sensor
   - power_sensor
+  - master_alarm
+  - failed_alarm
+ uses:
```
[the remark this answers is not in the document]
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml]: - run_command
  - run_status 
have been deleted as recommended...

**Comment by mschulze17** [comment]
@tasodorff back in your court

File: ontology/yaml/resources/HVAC/entity_types/FAN.yaml [hunk 2]
```diff
@@ -166,6 +174,15 @@ FAN_SS_DWI:
...
   - dishwasher_run_status
 
+FAN_STM:
+  description: "Basic fan with supply air temperature monitoring."
+  is_canonical: true
+  implements:
+  - FAN
+  - STM
+  uses:
+  - supply_air_temperature_sensor
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/FAN.yaml:184]: You don't need to define this field explicitly if you are implementing STM.

**Review by tasodorff — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/FAN.yaml (hunk 2 above)
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/FAN.yaml:184]: Removed Master_alarm and failed_alarm
- supply_air_temperature_sensor removed from uses of fan_stm

File: ontology/yaml/resources/HVAC/entity_types/FAN.yaml [hunk 3]
```diff
@@ -166,6 +174,15 @@ FAN_SS_DWI:
   uses:
   - dishwasher_run_status
 
+FAN_STM:
```
[the remark this answers is not in the document]
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/FAN.yaml:177]: added SS in the canonical type

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 4]
```diff
@@ -471,6 +471,7 @@ SS:
   - current_sensor
   - power_sensor
 
+
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:474]: Please revert this if its just whitespace.

File: ontology/yaml/resources/METERS/entity_types/EM.yaml [hunk 5]
```diff
@@ -20,6 +20,7 @@ EM_PWM:
   - EM
   - PWM
 
+
```
Comment (tasodorff) [inline ontology/yaml/resources/METERS/entity_types/EM.yaml:23]: Please revert if this is just whitespace.

**Review by tasodorff — APPROVED** [review]
Approval pending whitespace deletion.

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 4 above)
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:474]: This is just a white space...

File: ontology/yaml/resources/METERS/entity_types/EM.yaml (hunk 5 above)
Comment (RathishKJ) [reply ontology/yaml/resources/METERS/entity_types/EM.yaml:23]: This is just a white space...

**Review by tasodorff — CHANGES_REQUESTED** [review]
please revert whitespace

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 4 above)
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:473]: white space deleted

File: ontology/yaml/resources/METERS/entity_types/EM.yaml (hunk 5 above)
Comment (RathishKJ) [reply ontology/yaml/resources/METERS/entity_types/EM.yaml]: white space deleted

**Review by tasodorff — APPROVED** [review]

**Pull request merged**